### ISSUE TITLE
Add EventBridge target support in EventBridge rules

### ISSUE THREAD

**Issue by jesmiatka** [localstack/localstack#3352]
# Type of request: This is a ...

[ ] bug report
[x ] feature request

# Detailed description

Currently localstack does not support the target 'events' in EventBridge targets. This is supported by AWS and should be supported by localstack as well.

Localstack logs a warning:
https://github.com/localstack/localstack/blob/master/localstack/services/events/events_starter.py#L103

Code seems self explanatory, but can add a localstack setup if required.

## Expected behavior

Message is passed to other EventBridge target.

## Actual behavior

Warning is logged in console.

2020-12-14T10:27:19:WARNING:localstack.services.events.events_starter: Unsupported Events target service type "events"

# Steps to reproduce

## Command used to start LocalStack

```
awslocal events create-event-bus \
    --name bus-1

awslocal events create-event-bus \
    --name bus-2

awslocal events put-rule \
    --name "rule-bus-1" \
    --event-bus-name "bus-1"

awslocal events put-targets \
    --rule "rule-bus-1" \
    --event-bus-name "bus-1" \
    --targets "Id"="bus-2-rule-target","Arn"="arn:aws:events:us-east-1:000000000000:event-bus/bus-2"

awslocal events put-events --entries '[{"DetailType": "Test", "Source": "aws.test", "Detail": "{ }", "EventBusName": "bus-1"}]'
```

Running this script will trigger the warning:
`2020-12-16T08:29:26:WARNING:localstack.services.events.events_starter: Unsupported Events target service type "events"`

**Comment by usmangani1** [comment]
@jesmiatka  Thanks for reporting the issue. Can you please help us with sample aws cli commands / script using which we can reproduce this ? This will help us resolve the issue a bit sooner.Thanks.

**Comment by jesmiatka** [comment]
Hi @usmangani1,

I've added awslocal scripts in the initial issue. This will trigger the warning. 

You can also extend it by adding a SQS behind `bus-2`. In that case the SQS will not be filled, because the event is not pushed from `bus-1` to `bus-2`.

**Comment by whummer** [comment]
Thanks for reporting @jesmiatka . This should be fixed in #3602 - can you please pull the `latest` Docker image and give it another try? Please report here if the problem persists. Thanks!
